src/chapters/cloudfront/cloudfront-fundamentals.js

```diff
@@ -0,0 +1,124 @@
+
+## What We'll Accomplish Today
+
+By the end of this module, your Rock of Ages application will be:
+- Served through a global content delivery network
+- Available with HTTPS security
+- Optimized for speed with automatic file compression
+- Ready to handle visitors from anywhere in the world
+
+Let's start by understanding what CloudFront is and why it's valuable for your website.
+
+## What is CloudFront?
+
+Amazon CloudFront is a **Content Delivery Network (CDN)** - a system of distributed servers that deliver web content to users based on their geographic location. 
+
+Your S3 bucket stores your website's files in a single region (us-east-2/Ohio). Without CloudFront, every visitor to your site connects to that Ohio region to get your content, regardless of whether they're in California, Canada, or Croatia.
+
+CloudFront improves this by creating copies of your website at "edge locations" around the world. Visitors connect to the edge location nearest to them, significantly reducing the distance (and time) it takes for your content to reach them.
+
+## Why Your Website Benefits from CloudFront
+
+Your S3 website is already working, so why add CloudFront? Here's why CloudFront transforms your basic S3 website into a professional, production-ready application:
+
+### The Key Benefits of CloudFront
+
+Let's examine how CloudFront specifically enhances your website:
+
+**1. Improved Performance Through Proximity**
+
+Without CloudFront, a user in Tokyo accessing your S3 bucket in Ohio experiences a significant delay as their request travels around the world and back. With CloudFront, that same user connects to a Tokyo edge location with minimal latency.
+
+**Real-world example**: If a user in Tokyo and a user in New York both visit your website, the Tokyo user gets content from an Asian edge location while the New York user gets it from a North American edge location - both experiencing fast local performance even though your S3 bucket is only in the us-east-2 (Ohio) region.
+
+**2. Content Caching**
+
+CloudFront doesn't just provide a shorter path to your content - it also intelligently caches it:
+
+- When the first user requests your homepage
+- CloudFront retrieves it from your S3 bucket
+- CloudFront stores (caches) a copy at the edge location
+- When subsequent users request the same page, CloudFront serves it directly from the edge location
+- Your S3 bucket handles fewer requests, improving efficiency and reducing costs
+
+**3. Enhanced Security**
+
+S3 website endpoints only support HTTP, which modern browsers flag as "Not Secure." CloudFront automatically provides HTTPS support, giving your site the security that users expect from professional websites.
+
+**4. Global Reliability**
+
+If a region experiences issues, CloudFront can route users to other healthy regions. This built-in redundancy helps maintain availability during certain types of AWS outages.
+
```

Comment:
Lets also add something about price optimization. Requests through cloudfront cost less than requests straight to s3